CMakeLists.txt

```diff
@@ -1489,25 +1492,41 @@ if(ENGINEPRIME)
     target_link_libraries(mixxx-lib PUBLIC DjInterop::DjInterop)
   else()
     # Fetch djinterop sources from GitHub and build them statically.
-    message(STATUS "Linking statically to libdjinterop fetched from GitHub")
+
+    # On MacOS, Mixxx does not use system SQLite, so we will use libdjinterop's
+    # embedded SQLite in such a case.
+    if (APPLE)
+      message(STATUS "Building libdjinterop sources (with embedded SQLite) fetched from GitHub")
+      set(DJINTEROP_SYSTEM_SQLITE OFF)
+    else()
+      message(STATUS "Building libdjinterop sources (with system SQLite) fetched from GitHub")
+      set(DJINTEROP_SYSTEM_SQLITE ON)
+    endif()
 
     set(DJINTEROP_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib/libdjinterop-install")
+    set(DJINTEROP_LIBRARY "lib/${CMAKE_STATIC_LIBRARY_PREFIX}djinterop${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
Specifying `lib` here follows the pattern used for keyfinder, and should resolve the issue seen by @uklotzde [here](https://github.com/mixxxdj/mixxx/pull/2753#issuecomment-703912738).

Reply:
Seems reasonable since CMAKE_INSTALL_LIBDIR refers to the standard system location while the local builds use just "lib".